src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/Inputs/SARIF2004.OptimizeFileSize_Invalid.sarif

```diff
@@ -43,6 +43,36 @@
               }
             }
           ]
+        },
+        {
+          "ruleId": "TEST1001",
+          "ruleIndex": 0,
+          "message": { "text": "Bad result: artifacts array does not provide any additional useful information." },
```

Comment:
>artifacts array does not provide any additional useful information [](start = 44, length = 66)

Comment is wrong. This result is bad because `analysisTarget` is the same as one of the `locations`.